writeups/quic-bit-grease.txt

```diff
@@ -0,0 +1,175 @@
+As required by RFC 4858, this is the current template for the Document
+Shepherd Write-Up. Changes are expected over time.
+
+This version is dated 1 November 2019.
+
+(1) What type of RFC is being requested (BCP, Proposed Standard, Internet
+Standard, Informational, Experimental, or Historic)? Why is this the proper type
+of RFC? Is this type of RFC indicated in the title page header?
+
+Proposed Standard. This is the proper type for a simple extension to the use of
+the QUIC wire image.
+
+(2) The IESG approval announcement includes a Document Announcement Write-Up.
+Please provide such a Document Announcement Write-Up. Recent examples can be
+found in the "Action" announcements for approved documents. The approval
+announcement contains the following sections:
+
+Technical Summary:
+
+The "wire image" of QUIC packets is described in QUIC invariants (RFC 8999).
+Little about this image is visible to passive observers. One field, the
+so-called QUIC bit, resides at the second-to-most significant bit of the first
+byte of every QUIC packet. RFC 9000 requires the QUIC bit to be set to a value
+of 1, with the purpose being to allow QUIC traffic to be easily distinguisable
+from some other UDP-based traffic when they share a UDP port. This document defines a
+behavioural extension, negotiated using transport parameters, that permits
+endpoints to use any value for this bit. The intention of this
+extension is to exercise variation in the wire image in order to support future
+QUIC changes before systems ossify.
```

Comment:
maybe "exercise variation in the wire image (i.e. "GREASE" the QUIC bit) to prevent its ossification.", just to make a quick connection to the name of the bit.